docs/conf.py

```diff
@@ -93,7 +98,10 @@
 # further.  For a list of options available for each theme, see the
 # documentation.
 #
-# html_theme_options = {}
+
+# Show the version number in the navigation bar
```

Comment:
Version doesn't seem to be printed anywhere in local doc build. Seems like how versioning is handled may have changed: https://stackoverflow.com/a/79192065

and in particular, that it may now be only via a version selector that is only available if docs are hosted by RTD. Guess we could merge and see if this is the case (?)

I think we want the version information printed somewhere. One option would be to manually print the version somewhere, by putting `:Version: |version|` into the rst